.github/workflows/create_release.yml

```diff
@@ -74,22 +96,75 @@ jobs:
           draft: true
           prerelease: false
 
-      - name: Upload tar
+      - name: Upload core tar
         uses: actions/upload-release-asset@v1
```

Comment:
This action is unmaintained and I guess it's a good time to switch to a more recent variant of this.